src/main.js

```diff
@@ -6,6 +6,18 @@ var fs = require('fs');
 
 require('crash-reporter').start();
 
+// Load persisted state.
+var state = {};
+var statePath = path.join(app.getDataPath(), "state.json");
+try {
+  state = JSON.parse(fs.readFileSync(statePath, 'utf8'));
+} catch(e) {
+}
```

Comment:
`fs.readFileSync`doesn't specify which exception types are thrown. Also, I didn't know that you could use conditional catch clauses in JS, and MDN says they are non-standard.

We can think about logging caught exceptions to console, though this seems to be a very rudimentary form of error handling, and this block is not very critical. If we want to log things, I think we should use some kind of logging framework, though I haven't done any node development before, so I don't know what's usually done.